src/index.js

```diff
@@ -310,6 +340,8 @@ export default function (babel) {
         // TODO: handle declarations.length > 1
         // TODO: add linting to discourage
         targetPath.insertAfter(t.returnStatement(targetPath.node.declarations[0].id));
+      } else if (targetPath.isFunctionDeclaration()) {
```

Comment:
You might need something here to deal with `NamedArrowDeclaration`s. I did in my comprehension code.